packages/prisma/schema.prisma

```diff
@@ -686,6 +686,7 @@ model OrganizationSettings {
   allowSEOIndexing                    Boolean    @default(false)
   orgProfileRedirectsToVerifiedDomain Boolean    @default(false)
   disablePhoneOnlySMSNotifications    Boolean    @default(false)
+  orgAutoJoinOnSignup                 Boolean    @default(true)
```

Comment:
Since this feature has been default true since the beginning and users rely on this, it should be opt out.